packages/styleUtils/layout/gridList/style.ts

```diff
@@ -7,7 +7,9 @@ export const gridColumnTemplate = columnCount => {
           acc[breakpoint] = `repeat(${columnCount[breakpoint]}, 1fr)`;
           return acc;
         }, {})
-      : `repeat(${columnCount}, minmax(0, 1fr))`;
+      : columnCount
+      ? `repeat(${columnCount}, minmax(0, 1fr))`
+      : `repeat(auto-fill, minmax(min(320px, 100%), 1fr));`;
```

Comment:
could this have backwards compatibility issues?

Reply:
My thought was that this here protects us from breaking anything where the columnCount was explicitly set to a single number before. If we don't set any columnCount, it will default to the last part which should work in a lot of scenarios.

Reply:
okay cool!